samcli/local/docker/utils.py

```diff
@@ -85,7 +85,7 @@ def is_docker_reachable(docker_client):
     if platform.system() == "Windows":
         import pywintypes  # pylint: disable=import-error
```

Comment:
question: Is this suppression still required? Noticed that it was removed in `test_manager.py` but remains here.

Reply:
Ah, saw the commit where you had added it back, so I'm guessing this is still required 😃

Reply:
Indeed, Pylint is not great at working with stubs yet. While this wasn't an issue locally, there could be a bunch of different factors why it's different on the CI. And opted to just add the suppression back, rather than waste time trying to figure this specific oddity or start updating too many dependencies.